SWLOR.Game.Server/Feature/AbilityDefinition/Devices/IonGrenadeAbilityDefinition.cs

```diff
@@ -93,8 +102,9 @@ private void IonGrenade2()
             _builder.Create(FeatType.IonGrenade2, PerkType.IonGrenade)
                 .Name("Ion Grenade II")
                 .Level(2)
-                .HasRecastDelay(RecastGroup.IonGrenade, 30f)
-                .HasActivationDelay(1f)
+                .HasRecastDelay(RecastGroup.IonGrenade, 24f)
+                .HasRecastDelay(RecastGroup.Grenades, 6f)
```

Comment:
You can only have one recast group on a single ability. There shouldn't be a general-purpose "Grenades" group anymore. It should be based on the Ion Grenade recast group now. Please remove.